catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -182,15 +188,25 @@ public CreateResponse create(
 
       createResponse = doCreate(createRequest);
     } catch (IngestException e) {
+      INGEST_LOGGER.debug("Ingest error", e);
+      ingestError = e;
       rollbackStorage(createStorageRequest);
       throw e;
     } catch (IOException | RuntimeException e) {
+      ingestError = e;
       rollbackStorage(createStorageRequest);
+      INGEST_LOGGER.debug(
+          "Unable to store products for request: {} ... {}", streamCreateRequest.getId(), e);
       throw new IngestException(
           "Unable to store products for request: " + streamCreateRequest.getId(), e);
 
     } finally {
       opsStorageSupport.commitAndCleanup(createStorageRequest, tmpContentPaths);
+      if (createRequest != null && ingestError != null && INGEST_LOGGER.isInfoEnabled()) {
+        INGEST_LOGGER.info("Error on resource ingest. {}", ingestError);
+      }
+      INGEST_LOGGER.info(
+          "Resource ingest has completed for resource id: {}.", streamCreateRequest.getId());
```

Comment:
✏️ It'd be more valuable to admins/users if we logged the filename instead of the id.

Reply:
fixed